tests/test_partisn.py

```diff
@@ -139,6 +138,10 @@ def test_get_coord_sys_1D():
 def test_get_coord_sys_2D():
     """Test the _get_coord_sys function for a 2D mesh.
     """
+
+    if not HAVE_PYMOAB:
```

Comment:
There are a few functions, `write_partisn_input*`, which use the mesh module but do not have a similar skip block. These are probably called by other functions which are already skipping properly if PyMOAB is unavailable, but it would probably be good to include these skips there as well to avoid someone else running into mesh problems in the future.

Reply:
I wasn't sure which pattern to use... I went with the pattern of skipping in the actual test rather than skipping in the function that implements the test, but I suppose if that infrastructure changes, this could get lost.